_dev/src/ts/pages/UpdatePageVersionChoice.ts

```diff
@@ -90,6 +90,11 @@ export default class UpdatePageVersionChoice extends StepPage {
   };
 
   #saveForm = async () => {
+    api.currentRequestAbortController?.abort();
```

Comment:
What do you think about introducing a new method?

```suggestion
    api.abortCurrentCall();
```